src/prometheus_alert_rules/alertmanager_notifications_failed.rule

```diff
@@ -1,5 +1,5 @@
 alert: AlertmanagerNotificationsFailed
-expr: alertmanager_notifications_failed_total{integration=~".*"} != 0
+expr: rate(alertmanager_notifications_failed_total{integration=~".*"}[5m]) > 0
```

Comment:
**nitpick:** an `increase` would probably convey intent better.
Obviously, arithmetically the two are equivalent when the threshold is `0`.